src/Microsoft.AspNetCore.Server.IIS/HttpProtocol.cs

```diff
@@ -83,9 +84,9 @@ public HttpProtocol(PipeFactory pipeFactory, IntPtr pHttpContext)
 
                 // TODO: Read this from IIS config
                 PathBase = string.Empty;
-                Path = Encoding.ASCII.GetString(pHttpRequest->Request.pRawUrl, pHttpRequest->Request.RawUrlLength);
+                Path = Encoding.ASCII.GetString((byte*)(pHttpRequest->Request.pRawUrl), pHttpRequest->Request.RawUrlLength);
```

Comment:
We should change the code in http.sys to expose byte* instead of sbyte*. I'm not even sure why it's that way in the first place.

Reply:
I already asked that in HttpSys. Will change once addressed.